tox.ini

```diff
@@ -0,0 +1,57 @@
+# tox.ini
+# Runs docker-compose tests. Also creates a development virtualenv.
+# Usage:
+#   tox # runs the test suite
+#   tox -e develop $ create the dev virutalenv
+
+[pypi]
+# Use internal pypi with pre-build wheels for ubuntu
+index = https://registry.affirm-stage.com/artifactory/api/pypi/pypi/simple
+
+[tox]
+envlist = docker-compose, flake8
+minversion = 2.3
+indexserver =
+    default = {[pypi]index}
+# skip setup.py
+skipsdist = True
+
+[testenv]
+basepython = python2.7
+setenv =
+    PIP_INDEX_URL = {[pypi]index}
+    PYTHONIOENCODING = utf-8
+deps =
+  -rtox-requirements.txt
+# for consistency across installations
+sitepackages = False
+whitelist_externals =
+    bin/rm
+    bin/mkdir
+    bin/cp
+    bash
+
+[testenv:develop]
+envdir = .venv
+usedevelop = True
+
+[testenv:docker-compose]
+commands =
+    # xml test results files go here
+    rm -rf build/docker-compose
+    mkdir -p build/docker-compose
+    cp conf/development.env.example conf/development.env
+    bash -euc "docker-compose build"
+    bash -euc "docker-compose up -d"
+    # wait for the server to start up
+    bash -euc "sleep 20"
```

Comment:
perhaps we can do a smart wait, instead of a sleep. e.g. we do this for platform docker images https://github.com/Affirm/all-the-things/blob/master/build.sh#L176-L180 . the "jjjj/wait" image by Jim, waits until the connected container is accepting connections. https://github.com/jimjh/docker-wait